packages/clay-css/src/scss/variables/_multi-step-nav.scss

```diff
@@ -34,7 +34,7 @@ $data-multi-step-icon-before-content: unquote(
 	"'\\FEFF' attr(data-multi-step-icon)"
 ) !default;
 
-$multi-step-divider-bg: $gray-200 !default;
+$multi-step-divider-bg: $gray-300 !default;
```

Comment:
We should add this to `src/scss/atlas/variables/_multi-step-nav.scss` instead of changing it here.